bot/cogs/snakes.py

```diff
@@ -1,8 +1,10 @@
 # coding=utf-8
-import logging
+import logging, aiohttp, random, wikipedia
```

Comment:
No, you should be using separate lines, and they should be in alphabetical order.